clients/html/src/app/employer-details/employer-details.component.html

```diff
@@ -490,20 +490,24 @@ <h6 class="sub-heading-text">Dependent {{ j + 1 }}</h6>
             <!-- starts employee fields -->
             <fieldset class="household mb-3">
               <legend>
+                <h1 class="wave_hidden">Employee</h1>
+                <h2 class="wave_hidden">Employee</h2>
+                <h3 class="wave_hidden">Employee</h3>
+                <h4 class="wave_hidden">Employee</h4>
```

Comment:
Do we need this hidden tags?

Reply:
Yes, these are recommendations from the wave tool. We should follow these.